src/engraving/dom/note.h

```diff
@@ -296,9 +296,13 @@ class Note final : public EngravingItem
     Tie* tieBack() const { return m_tieBack; }
     void setTieFor(Tie* t) { m_tieFor = t; }
     void setTieBack(Tie* t) { m_tieBack = t; }
-    Note* firstTiedNote() const;
-    const Note* lastTiedNote() const;
-    Note* lastTiedNote() { return const_cast<Note*>(static_cast<const Note*>(this)->lastTiedNote()); }
+    Note* firstTiedNote(bool ignorePlayback = true) const;
```

Comment:
These new flags should be false by default.  firstTiedNote / lastTiedNote can be used during layout, and playSpanner() should not affect layout at all